test/api/handlers_test.go

```diff
@@ -950,6 +956,21 @@ func testUserFlow(t *testing.T, at *test.AccountsTester) {
 	if at.Cookie == nil {
 		t.Fatalf("Failed to extract cookie from request. Cookies found: %+v", r.Cookies())
 	}
+	// Make sure the response contains a valid JWT.
+	tk := r.Header.Get("Skynet-Token")
+	if _, err = jwt.ValidateToken(tk); err != nil {
+		t.Fatal("Missing or invalid token. Error:", err)
+	}
+	// Make sure we can make calls with this token.
+	at.Token = tk
+	c := at.Cookie
+	at.Cookie = nil
+	_, _, err = at.Get("/user", nil)
```

Comment:
I have a ticket for that (https://github.com/SkynetLabs/skynet-accounts/issues/128) but I'd prefer to do it as a F/U under that ticket, if you don't mind. Otherwise it will have to wait for https://github.com/SkynetLabs/skynet-accounts/pull/131 to be merged, at least, because that one modifies the return types of many handlers.

Reply:
Does it make sense to add explicit `GetCustom` that allows passing a cookie and a token?

Reply:
f/u is fine

Reply:
> Does it make sense to add explicit `GetCustom` that allows passing a cookie and a token?

My intention was to make them parameters of the methods and document that you can pass `nil`s.